### DISCUSSION TITLE
Is input-leap compatible with barrier?

### DISCUSSION THREAD

**Discussion by janevert** [input-leap/input-leap#1942] [Q&A] [answered]
Hi all,

Is input-leap compatible with barrier?

I intend to upgrade the barrier server on my Debian to input-leap. Or at least install it so I can test it. The goal is to switch from X to Wayland on Debian.
The single client is barrier 2.4.0 on a Windows machine (since it is company controlled it is pita to upgrade).
In case it is relevant, an ssh tunnel is used.

Thanks

**Comment by shymega** [comment, marked as answer]
_At this time_, yes. But I wouldn't advise it.

**Reply by DTS-And-Eaglepoint-Funding** [reply]
im using it together as i slowly migrate to input-leap everything is working

**Reply by shymega** [reply]
Sure, but you should be aware Input Leap may change the protocol. And Barrier has several differences to Input Leap. 

Barrier is also unmaintained, which adds to my stance.
